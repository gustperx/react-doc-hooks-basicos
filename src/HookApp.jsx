export const HookApp = () => {
  return (
    <>
      <h1 className="text-2xl text-orange-300">Hola Mundo</h1>
    </>
  );
};
