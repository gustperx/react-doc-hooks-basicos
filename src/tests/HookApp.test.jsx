import { shallow } from "enzyme";
import { describe, expect, test } from "vitest";
import { HookApp } from "../HookApp";

describe("Pruebas en HookApp", () => {
  test("debe de hacer match con el snapshot", () => {
    const wrapper = shallow(<HookApp />);

    expect(wrapper).toMatchSnapshot();
  });
});
