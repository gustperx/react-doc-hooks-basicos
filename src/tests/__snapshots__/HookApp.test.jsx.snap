// Vitest Snapshot v1

exports[`Pruebas en HookApp > debe de hacer match con el snapshot 1`] = `
<Fragment>
  <h1
    className="text-2xl text-orange-300"
  >
    Hola Mundo
  </h1>
</Fragment>
`;
